pilot/pkg/networking/core/v1alpha3/route/retry/retry.go

```diff
@@ -60,7 +59,7 @@ var (
 func ToEnvoyConfig(in *networking.HTTPRetry) *route.RouteAction_RetryPolicy {
 	if in == nil {
 		// No policy was set, use a default.
-		return getDefaultRetryPolicy()
+		return DefaultPolicy()
```

Comment:
I think we should use `defaultPolicy` directly in package `retry`.  DefaultPolicy is for other pkgs

Reply:
The reason I'm using `DefaultPolicy` everywhere is to ensure that we're returning a copy.

Reply:
Not that I'm aware of. But this prevents modification from affecting other clusters.